pkg/util/logger_windows.go

```diff
@@ -0,0 +1,134 @@
+package util
+
+import (
+	"errors"
+	"fmt"
+	"runtime"
+	"strings"
+
+	"github.com/go-kit/kit/log/level"
+
+	"github.com/go-kit/kit/log"
+	"github.com/weaveworks/common/server"
+	el "golang.org/x/sys/windows/svc/eventlog"
+)
+
+// Default name for the Grafana Agent under Windows
+const ServiceName = "Grafana Agent"
+
+//NewWindowsEventLogger creates a new logger that writes to the event log
+func NewWindowsEventLogger(cfg *server.Config) *Logger {
+	l := Logger{makeLogger: makeWindowsEventLogger}
+	if err := l.ApplyConfig(cfg); err != nil {
+		panic(err)
+	}
+	return &l
```

Comment:
Should this call `newLogger(cfg, makeWindowsEventLogger)` instead of re-implementing it?